intg/src/main/java/org/apache/atlas/model/typedef/AtlasStructDef.java

```diff
@@ -280,6 +280,8 @@ public static class AtlasAttributeDef implements Serializable {
         public static final String    ATTRDEF_OPTION_APPEND_ON_PARTIAL_UPDATE = "isAppendOnPartialUpdate";
         private final String          STRING_TRUE                             = "true";
         private final String          MULTIFIELDS                             = "multifields";
+        private Boolean               isIndexCreated                          = true;
```

Comment:
why default value for `isIndexCreated` is set to true ? In general, let the flag be `false`, since index is not created by default, and only through certain operation, these flags should be set `true`